### PR TITLE
feat(e2e): kustomize configs for e2e env

### PR DESCRIPTION
**Pull request by edgarulg**

### CODE REVIEW

**Review by christosarvanitis — CHANGES_REQUESTED** [review]
Is this the correct repo for storing the testing instances configuration? This is a public repo that potentially a customer can utilize

File: terraform/spin-prerequisites/main.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
+terraform {
+  backend "s3" {
+  }
+}
+
+provider "vault" {
```
Comment (christosarvanitis) [inline terraform/spin-prerequisites/main.tf:6]: Remove L6:L14 & Remove L18:L20

File: terraform/tfstate-backend/main.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+module "s3_bucket" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+  bucket = "internal-prod-spinnaker-sandbox-integrations-tfstate-backends"
```
Comment (christosarvanitis) [inline terraform/tfstate-backend/main.tf:7]: Change the bucket name

File: terraform/tfstate-backend/terraform.tfstate [hunk 3]
```diff
@@ -0,0 +1,230 @@
+{
```
Comment (christosarvanitis) [inline terraform/tfstate-backend/terraform.tfstate:1]: Remove this file and after apply commit the new terraform.tfstate

File: terraform/tfstate-backend/variables.tf [hunk 4]
```diff
@@ -0,0 +1,7 @@
+variable "aws_region" {
+  default = "us-west-2"
+}
+
+variable "team" {
+  default = "managed-services"
```
Comment (christosarvanitis) [inline terraform/tfstate-backend/variables.tf:6]: You can have dynamo as default

File: dinghyfile [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+     "github_account": "github",
+     "github_api": "https://api.github.com/repos/armory-io/release-patches"
+  },
+  "pipelines": [
+     {{ local_module "modules/pipelines/pipeline.deploy-destroy-spinnaker.module"
+         "k8s_account" "kubernetes-eks-prod"
+         "team" "dynamo"
+         "namespace" "other-ephemeral-spinnaker-sandbox"
+         "pipeline" "create/destroy spinnaker instance"
+         "spinnaker_gitrepo_account" "gitrepo-armory"
```
Comment (christosarvanitis) [inline dinghyfile:20]: Remove the variables that are defined in the globals

**Comment by edgarulg** [comment]
I closed this PR in favor of https://github.com/armory-io/armory-enterprise-testing/pull/1

**Pull request closed without merging**